### PR TITLE
Copy URL automatically to clipboard and get a confirmation on the paste page...

### PR DESCRIPTION
**Pull request by aruppert**
...only if you pasted the text yourself.

Screenshot 2019-12-06 at 14 13 46

### CODE REVIEW

**Review by laurapauly — COMMENTED** [review]
lgtm :)

File: client/src/components/Confirmation.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import { keyframes } from "styled-components";
+
+export const flyFromRight = keyframes`
```
Comment (lmachens) [inline client/src/components/Confirmation.js]: `Confirmation` is a copy of `Alert`. I would suggest to add a property to change the color.

File: client/src/App.js [hunk 2]
```diff
@@ -3,13 +3,29 @@ import GlobalStyles from "./GlobalStyles";
...
+        <Confirmation>
+          <div>🎉🎉🎉</div>URL copied to your clipboard!
+        </Confirmation>
+      )}
       <Router>
-        <Route exact path="/" component={Home} />
+        <Route
+          exact
+          path="/"
+          component={() => <Home isPasteCreator={isPasteCreator} />}
```
Comment (lmachens) [inline client/src/App.js]: `isPasteCreator` is misleading because I would suggest having a boolean instead of a function

Comment (lmachens) [reply client/src/App.js]: `onPaste` maybe

File: client/src/pages/Paste.js [hunk 3]
```diff
@@ -34,7 +45,7 @@ export default function Paste({ match }) {
           <CreatedAt date={new Date(paste.createdAt)}>
             {new Date(paste.createdAt).toDateString()}
           </CreatedAt>
-          <PasteArea>{paste.value}</PasteArea>
+          <PasteArea onChange={copyURL()}>{paste.value}</PasteArea>
```
Comment (lmachens) [inline client/src/pages/Paste.js]: `copyURL` is executed on every render. Thus, `onChange` is undefined

**Review by lmachens — CHANGES_REQUESTED** [review]

**Pull request merged**